docs/_index.md

```diff
@@ -3,32 +3,32 @@ title: "Welcome to Knative"
 linkTitle: "Documentation"
 weight: 10
 type: "docs"
+showlandingtoc: "false"
 aliases:
    - /docs/concepts/overview.md
 ---
 
-The Knative project provides a set of [Kubernetes controllers](https://kubernetes.io/docs/concepts/architecture/controller/) that help to simplify developer and administrator experiences on [Kubernetes](https://kubernetes.io) clusters, by introducing an event-driven, serverless architecture.
+The Knative project provides a set of [Kubernetes](https://kubernetes.io) components that introduce event-driven, serverless capabilities for Kubernetes clusters.
```

Comment:
"event driven and serverless" -- you can choose either, but don't need to take both.

Reply:
Good point